account_reconcile_restrict_partner_mismatch/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Account Partner Reconciliation',
```

Comment:
Should ibe closer to the real name of the module : Account Reconcile Restrict Partner Mismatch